app/src/main/java/at/techbee/jtx/ui/detail/DetailsTopAppBar.kt

```diff
@@ -54,6 +56,21 @@ fun DetailsTopAppBar(
 ) {
 
     var textFieldText by rememberSaveable { mutableStateOf("") }
+    var previousText by rememberSaveable { mutableStateOf(emptyPreviousText) }
+
+    fun onSubtaskDone(value: String) {
```

Comment:
@t-h-e careful here with the naming, this now indicates that the subtask itself is done ;-)